ceph-dashboard-pull-requests/config/definitions/ceph-dashboard-pull-requests.yml

```diff
@@ -30,9 +30,10 @@
           allow-whitelist-orgs-as-admins: true
           org-list:
             - ceph
+          white-list-labels: "dashboard"
```

Comment:
Does this mean that if there is no `dashboard` label in the PR, the build won't trigger? Then this would be a narrowing down, if someone missed to label a dashboard PR.

Reply:
Yes, that would mean the Job won't be triggered for PRs without the ``dashboard`` label.
That's true, but usually we do assign the dashboard label to our PRs and I don't know how or what kind of other 'filter' I could use to only run this Job for dashboard-related PRs.